henperi/Fast-Food-Fast
# Host and Render User Interface and all static files on heroku

#### What does this PR do?
- This PR renders the user interface on heroku

#### Description of Task to be completed?
- Use UI folder as express.static assets folder
- Clone existing UI into the server folder

#### How should this be manually tested?
- Visit ```https://api-fast-food-fast.herokuapp.com/``` to see the interface 
- If local, Visit ```localhost:5000```

#### What are the relevant pivotal tracker stories?
#161342613
